### PR TITLE
🌱 Makefile: add crds target

### PR DESCRIPTION
**Pull request by sttts**
`make crds` for much faster iteration.

Your PR description is *really* short. Your PR description is *really* short. Your PR description is *really* short. Your PR description is *really* short. Your PR description is *really* short. Your PR description is *really* short.

### CODE REVIEW

**Review by ncdc — APPROVED** [review]

**Comment by stevekuznetsov** [comment]
What's flaking? Do we have an issue open for it?

**Comment by sttts** [comment]
Rebased.

/approve

File: hack/update-codegen-crds.sh [hunk 1]
```diff
@@ -17,15 +17,13 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-set -o xtrace 
+# set -o xtrace
```
Comment (p0lyn0mial) [inline hack/update-codegen-crds.sh:20]: rm or bring back?

Comment (ncdc) [reply hack/update-codegen-crds.sh]: Restored

**Comment by ncdc** [comment]
Hit flake: #2389

**Comment by ncdc** [comment]
More flaking

/retest

**Comment by sttts** [comment]
Merged through #2510.

**Pull request closed without merging**